src/flux/task.coffee

```diff
@@ -169,6 +169,15 @@ TaskConfig =
 
       if allowed.indexOf(@_get(taskId).type) > -1 then true else false
 
+    getRelatedSections: (taskId) ->
+      _.chain(getSteps(@_steps[taskId]))
+        .pluck('chapter_section')
+        .compact()
+        .map( (cs) -> cs.join('.') )
+        .uniq()
+        .value()
+
+
```

Comment:
Great idea.  I had the `.map` in there mainly so the later `uniq` would compare properly. But it's really not needed though since `.uniq` also accepts a comparator function.